### ISSUE TITLE
build node.js executable

### ISSUE THREAD

**Issue by atsushieno** [atsushieno/mugene-ng#4]
I set up webpack.config.d for this purpose, but if  add  `binaries.executable()` at `kotlin { js { ... } }` section in `build.gradle.kts` build fails like:

```
Module 'fs' not found
Module 'kotlinx-coroutines-core' not found
```

This needs to be sorted out.

**Comment by atsushieno** [comment]
Another blocking issue in Kotlin itself: https://youtrack.jetbrains.com/issue/KT-28664 - this makes it impossible to build mugene module for both browser and node.

**Comment by atsushieno** [comment]
There is no way to work around missing `fs` on `browser()`, so we skip `browser()` so far.

`kotlinx-coroutines-core` issue is gone once we removed js-specific `implementation` section.

We don't do with `js(BOTH)` or `js(IR)` so far (i.e. no IR) and enabled `binaries.executable()` as well as headless tests using karma.

Closing this as it's so far non-issue anymore. This may occur in the future whenever we play around those JS generator modes or supported environment though.
